deployments/sdm-relay/templates/clusterrole.yaml

```diff
@@ -1,3 +1,5 @@
+{{- if .Values.strongdm.discoveryUsername }}
+
```

Comment:
Not quite. The health-check cluster role and role binding at the beginning of this file should always exist.